### PR TITLE
Picking bugfix and API improvements

### PR DESCRIPTION
**Pull request by jailln**
Some bugfixes and API modifications I had to make for Elevation measure widget #1881 (this PR contains the first commits of PR #1881 that do not relate to elevation measure widget.

fix(picking): fix picking on multiple layers
refacto(view): rename pickCoordinates to pickTerrainCoordinates
example(potree_25d_map): declare map projection code
feat(picking): Add distance and point position to returned object by points picking
feat(Coordinates): Add a setCrs method

These commits have already reviewed in #1881 , so if we can merge it quickly I would appreciate :)

### CODE REVIEW

File: src/Core/View.js [hunk 1]
```diff
@@ -1037,9 +1037,9 @@ class View extends THREE.EventDispatcher {
      * @param   {Coordinates}           [target]    The result will be copied into this {@link Coordinates}. If not
                                                     * specified, a new {@link Coordinates} instance will be created.
      *
-     * @returns {Coordinates}   The world {@link Coordinates} at the given view coordinates.
+     * @returns {Coordinates}   The world {@link Coordinates} of the terrain at the given view coordinates.
```
Comment (mgermerie) [inline src/Core/View.js]: Could you add the info that the output coordinates' crs defaults to `view.referenceCrs` if no `target` is given ? (Here or above, on `target` documentation).

**Review by mgermerie — COMMENTED** [review]
Your first commit ([fix(picking): fix picking on multiple layers](https://github.com/iTowns/itowns/pull/2004/commits/3f54221c1adf29b115144023b71eb220b5e06c1e)) is later canceled by your fourth commit ([feat(picking): Add distance and point position to returned object by …](https://github.com/iTowns/itowns/pull/2004/commits/6b860b538d9a9905ed2cbf481dd6ed64c485bdb8)). Moreover, what is done in the first commit is already done [here](https://github.com/iTowns/itowns/blob/c83bfcf74ceee1358d2070b1c90079900744086e/src/Core/Picking.js#L187).

File: src/Core/View.js (hunk 1 above)
Comment (jailln) [reply src/Core/View.js]: Done

**Comment by jailln** [comment]
> Your first commit ([fix(picking): fix picking on multiple layers](https://github.com/iTowns/itowns/pull/2004/commits/3f54221c1adf29b115144023b71eb220b5e06c1e)) is later canceled by your fourth commit ([feat(picking): Add distance and point position to returned object by …](https://github.com/iTowns/itowns/pull/2004/commits/6b860b538d9a9905ed2cbf481dd6ed64c485bdb8)). Moreover, what is done in the first commit is already done [here](https://github.com/iTowns/itowns/blob/c83bfcf74ceee1358d2070b1c90079900744086e/src/Core/Picking.js#L187).

Yes you're right I think I mixed things up while rebasing. I squashed those two commits.

**Review by mgermerie — APPROVED** [review]

**Pull request merged**